packages/core/src/utils/common.js

```diff
@@ -66,6 +66,9 @@ export function getMemberExressionTarget(node: MemberExpression): Node {
   return target;
 }
 
-export function ensureArray<T>(value: T | Array<T>): Array<T> {
+export function ensureArray<T>(value: T | Array<T> | null | undefined): Array<T> {
+  if (value === undefined || value == null) {
```

Comment:
One more thing, `value == null` convert to `value === null`

Reply:
oh mistyped here - fixed